pronto_mia/lib/ui/views/startup/startup_viewmodel.dart

```diff
@@ -44,4 +49,14 @@ class StartUpViewModel extends BaseViewModel {
       _navigationService.replaceWith(Routes.loginView);
     }
   }
+
+  Future<void> _handlePushNotifcations(bool isAuthenticated) async {
+    final notificationsAuthorized =

```

Comment:
- Can go within the if? A user does not need notifications as long as he is not authorized.